src/lazy/binary/raw/v1_1/type_descriptor.rs

```diff
@@ -55,56 +63,131 @@ impl Opcode {
     /// opcode + length code combination is illegal, an error will be returned.
     pub const fn from_byte(byte: u8) -> Opcode {
```

Comment:
🗺️ This method is run at compile time to compute a `u8`->`Opcode` cache. I've augmented it to also compute the `LengthType` and `OpcodeKind` for each byte.